jumpcloud-ADMU/Powershell/Form.ps1

```diff
@@ -143,92 +153,211 @@ function show-mtpSelection {
+                <TextBlock Name="lbl_orgNameTitle" HorizontalAlignment="Left" Margin="10,229,0,0" Text="Organization Name:" VerticalAlignment="Top" FontWeight="Normal"/>
+                <TextBlock Name="lbl_orgName" HorizontalAlignment="Left" Margin="124,229,0,0" Text="Not Currently Connected To A JumpCloud Organization" VerticalAlignment="Top" FontWeight="Normal"/>
+                <CheckBox Name="cb_forcereboot" Content="Force Reboot" HorizontalAlignment="Left" Margin="10,76,0,0" VerticalAlignment="Top" FontWeight="Normal" IsChecked="False"/>
+                <CheckBox Name="cb_installjcagent" Content="Install JCAgent" HorizontalAlignment="Left" Margin="10,36,0,0" VerticalAlignment="Top" FontWeight="Normal" IsChecked="False"/>
+                <CheckBox Name="cb_bindAsAdmin" Content="Bind As Admin" HorizontalAlignment="Left" Margin="118,56,0,0" VerticalAlignment="Top" FontWeight="Normal" IsChecked="False" IsEnabled="False"/>
+                <CheckBox Name="cb_leavedomain" ToolTipService.ShowOnDisabled="True" Content="Leave Domain" HorizontalAlignment="Left" Margin="10,56,0,0" VerticalAlignment="Top" FontWeight="Normal" IsChecked="False"/>
+                <CheckBox Name="cb_autobindjcuser" Content="Autobind JC User" HorizontalAlignment="Left" Margin="118,36,0,0" VerticalAlignment="Top" FontWeight="Normal" IsChecked="False" />
+                <Image Name="img_ckey_info" Height="14" Width="14" Visibility="Visible" ToolTip="The Connect Key provides you with a means of associating this system with your JumpCloud organization. The Key is used to deploy the agent to this system." Margin="147,124,309,122" />
+                <Image Name="img_ckey_valid" HorizontalAlignment="Left" Height="23" Margin="446,145,0,0" VerticalAlignment="Top" Width="14" Visibility="Hidden" ToolTip="Connect Key must be 40chars &amp; not contain spaces" />
+                <Image Name="img_apikey_info" HorizontalAlignment="Left" Height="14" Margin="124,177,0,0" VerticalAlignment="Top" Width="14" Visibility="Visible" ToolTip="Click the link for more info on how to obtain the api key. The API key must be from a user with at least 'Manager' or 'Administrator' privileges." RenderTransformOrigin="1.857,-1.066"/>
+                <Image Name="img_apikey_valid" HorizontalAlignment="Left" Height="23" Margin="446,198,0,0" VerticalAlignment="Top" Width="14" Visibility="Hidden" ToolTip="Correct error" />
+            </Grid>
         </GroupBox>
+
+        <!-- Migrate Button -->
+        <Button Name="bMigrateProfile" HorizontalAlignment="Left" VerticalAlignment="Top" Margin="834,546,0,0" Width="146" Height="26" IsEnabled="False" FontWeight="SemiBold" Content="Migrate Profile">
+            <Button.Resources>
+                <Style TargetType="{x:Type Border}">
+                    <Setter Property="CornerRadius" Value="3"/>
+                </Style>
+            </Button.Resources>
+        </Button>
```

Comment:
A few things with form:
![Screenshot 2024-07-12 at 1 38 14 PM](https://github.com/user-attachments/assets/15a4451e-9814-46ba-a742-4a401a18d2a2)

- The content should be aligned in a grid.
- The logo isn't aligned with the "system information" box
- The "Select a domain or Azure AD account to be migrated" box isn't right-aligned with "system migration options" box
- The "Migrate Profile" button should be aligned with the same space on the bottom of the app as the logo. I think we can clean up these lines here it's small stuff but it makes a difference.
- The "System Information" and "Select a domain or Azure AD account to be migrated" boxes should have the same spacing from the bottom of the other boxes to the migrate profile button. Let's call and I can explain this exactly.
- I also think the "Account Migration Information" and the "System Migration Options" should be flipped. There's something that I think will come up as a UI bug If we don't do this. I have a recording can show.